### PR TITLE
Add flags to include/exclude namespaces

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

**Comment by rbren** [comment]
Still haven't tested this in a real-world setting - will do that once the images build.

File: e2e/test.sh [hunk 1]
```diff
@@ -55,11 +59,15 @@ echo "********************************************************************"
 printf "\n\n"
 
 kubectl create ns goldilocks
+
 kubectl -n goldilocks apply -f /hack/manifests/dashboard/
 kubectl -n goldilocks apply -f /hack/manifests/controller/
 
-kubectl -n goldilocks wait deployment --timeout=60s --for condition=available -l app.kubernetes.io/name=goldilocks,app.kubernetes.io/component=dashboard
-kubectl -n goldilocks wait deployment --timeout=60s --for condition=available -l app.kubernetes.io/name=goldilocks,app.kubernetes.io/component=controller
+sleep $reinstall_wait
```
Comment (sudermanjr) [inline e2e/test.sh]: I'm not sure why this is necessary since we have the kubectl wait below.  Can you shed some more light on that?

Comment (rbren) [reply e2e/test.sh]: Whoops, not sure where that came from.

**Comment by rbren** [comment]
Done!

**Pull request merged**